portfolio/src/main/webapp/scripts/script.js

```diff
@@ -12,17 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
-}
+/* TODO:Format */
+function reload(){
+  window.open("https://8080-dot-12459592-dot-devshell.appspot.com/","_self");
```

Comment:
What is this link, exactly? My Google account does not have access to view it. If it's just meant to load the index.html page, can't that be accomplished by just referencing the local index.html file?

Reply:
Yes. I think this was a temporary link to my local app on port 8080 and I fixed in a latter branch's commit. Fixed.